tests/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 # SPDX-License-Identifier: MIT
 cmake_minimum_required(VERSION 3.15)
 
+option(BUILD_FFT_KERNEL      "Build DDC kernel for FFT" ON)
+option(BUILD_SPLINES_KERNEL  "Build DDC kernel for splines" ON)
+
```

Comment:
Seems to be redundant with the top-level directory `CMakeLists.txt`
```suggestion
```

Reply:
Ah, I don't think so, my understanding is that options are in the scope of current CMakeLists and this is the way to transmit it to tests/CMakeLists. I checked the behavior which seems correct